templates/appliance/index.html

```diff
@@ -123,21 +94,34 @@ <h2>Try before you Pi</h2>
 
   <div class="row">
     <div class="col-7">
-      <h2>App store easy</h2>
-      <p class="p-heading--four">An app or two for extra foo</p>
-      <p>Upgrade your appliance with extra apps. Try out a new autopilot. Add an extra game. Each appliance starts with just the right stuff for everyone. Add the sauce you like.</p>
-      <p>Enjoy the same <a class="p-link--external" href="https://snapcraft.io/store">huge app selection as Ubuntu</a>, or get a dedicated store with unique apps for that specific thing. Publishers will shape the store you see.</p>
-      <p>Ubuntu Appliances get apps and updates from our global backbone, just like millions of Ubuntu workstations, laptops and servers every day.</p>
+      <h2>
+        App store easy
+      </h2>
+      <p class="p-heading--four">
+        An app or two for extra foo
+      </p>
+      <p>
+        Upgrade your appliance with extra apps. Try a new autopilot. Enjoy the latest game. Each appliance starts with the essential stuff. Add the sauce you like.
+      </p>
+      <p>
+        Enjoy the same huge app selection as Ubuntu, or get a dedicated store with unique apps for your specific thing. Publishers curate the store you see.
```

Comment:
"huge app selection as Ubuntu" is a link to the snap store in the copy doc

Reply:
removed from the copy doc